pwnedapi/utils.py

```diff
@@ -1,10 +1,14 @@
+import json
 import tablib
-
 from pwnedapi.Password import Password
 from time import sleep
 from typing import Any
 
 
+# this override fixes an incompatability if ujson is installed
```

Comment:
Should we detect the presence of ujson in the system somehow and only then do this?